packages/ui-components/style/rjsfTemplates.css

```diff
@@ -130,7 +130,7 @@
   fill: var(--jp-ui-font-color0);
 }
 
-button.jp-ArrayOperationsButton.jp-mod-styled:disabled {
+.jp-ThemedContainer button.jp-ArrayOperationsButton.jp-mod-styled:disabled {
```

Comment:
Is this needed for buttons and other elements which already have `.jp-mod-styled`? There is a performance penalty for having all such elements be traversed up the DOM tree to check the container class.

This pertains to changes in here and
- `packages/running/style/base.css`,
- `packages/ui-components/style/styling.css`
- and some (not all) in `packages/application/style/buttons.css`

Reply:
That was one of my doubt. So I'll revert those changes to limit the performance penalty

Reply:
Suggestions applied in ae8d105